wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py

```diff
@@ -198,6 +206,22 @@ def _file_url(
                 api.settings("base_url"), entity_name, md5_hex
             )
         elif storage_layout == StorageLayout.V2:
+            if self._api._check_server_feature_with_fallback(
```

Comment:
@estellazx quick sanity check - is this the only place where we build the download url for a single artifact file?